### DISCUSSION TITLE
PublicUrl

### DISCUSSION THREAD

**Discussion by SmsS4** [seaweedfs/seaweedfs#4509] [Q&A]
Hi  
I saw `-publicUrl` flag in `weed volume` but I don't really understand it's usage.  
Does master, shell or filer use publicUrl?  
Thanks.

**Comment by n9** [comment]
I have found two usages:
- in the master UI: where the `publicUrl` is displayed as a link
- in the mount options: `-volumeServerAccess=publicUrl` which then uses `publicUrl`:
  https://github.com/seaweedfs/seaweedfs/blob/c09100f928466116fa648474417e30540026d8a4/weed/command/mount.go#L63
